CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Added
 
+- Support modification of task priorities via `pueue edit --priority/-o` and `pueue restart --edit-priority/-o` [#449](https://github.com/Nukesor/pueue/issues/449)
```

Comment:
Minor merge conflict.

I forgot to update the development branch after the latest patch releases on master again. It's an easy fix though :)